pkg/k8s/common.go

```diff
@@ -321,3 +322,47 @@ func getPortForwardUrl(config *rest.Config, namespace string, podName string) *u
 		Host:   hostIp,
 	}
 }
+
+func watchForReady(deploymentName *string, readyChan chan<- bool) {
+	go func() {
+		for {
+			deployment, err := deploymentsClient.Get(context.Background(), *deploymentName, metav1.GetOptions{})
+			if err != nil {
+				log.Error(err)
+				os.Exit(1)
+			}
+			msg, done, err := deploymentStatus(deployment)
+			print(msg)
+			if done {
+				readyChan <- true
+				return
+			} else if err != nil {
+				log.Error(err)
+				os.Exit(1)
```

Comment:
Sorry. I based the error handling off of the existing waitForReady method. 

Not sure how you are envisioning this since. I opted to repurpose readyChan to indicate Not Ready. One alternative I considered was passing the sig channel all they way through the code.